### PR TITLE
Improve code readability

### PR DESCRIPTION
**Pull request by Nopraz**
The button Is enabled only when the game is playing

### CODE REVIEW

File: src/components/Board.vue [hunk 1]
```diff
@@ -102,27 +107,19 @@
...
-                this.resetSnake();
-
-                this.squares[this.getFoodRandomCoords()] = SNAKE.FOOD;
-
+                this.resetGame();
+                this.run();
+            },
+            reset(){
+                this.playSound('gameStart');
+                if (this.gameRunning == null || this.gameRunning === false) {
```
Comment (Matrix159) [inline src/components/Board.vue]: This can be turned into if (!this.gameRunning) as it will check for undefined, null, or false

**Comment by Lloople** [comment]
Hi! Thank you for taking the time to make this PR!

I'm sorry, but I'm not interested in adding a `reset` button to the game by now.

However, what I liked is that you moved the logic to a method called `resetGame` 👌.

Would you mind removing the new button, the `reset` function and the new comments and leave this new method you've created? You can close this PR and make a new one if you want, or make the changes in this one. As you prefer 👍

File: src/components/Board.vue [hunk 2]
```diff
@@ -303,6 +300,24 @@
...
+
+                // Resets grid, snake position, and snake movement variables back to their defaults
+                this.squares = this.resetGrid();
+                this.direction = DIRECTION.RIGHT;
+                this.snakeHead = SNAKE.HEAD_START;
+                this.snakeBody = SNAKE.BODY_START;
+                this.canChangeDirection = true;
+                this.resetSnake();
+
+                this.squares[this.getFoodRandomCoords()] = SNAKE.FOOD;
```
Comment (Lloople) [inline src/components/Board.vue]: I like what you did here, but please undo all the other changes and remove the comments as they are no needed to understand the code 🙏

**Comment by Nopraz** [comment]
Hi ! just did the changes you requested, i removed the reset button and his function but leaved the resetGame() function without the comments

**Pull request merged**